### DISCUSSION TITLE
Hearing Microphone audio trough headset

### DISCUSSION THREAD

**Discussion by BlizzieM** [alvr-org/ALVR#1780] [Q&A]
Hey!
I am running the appimage version of the software under linux and I keep hearing my voice trough the headhpones on my Quest 2. This is something I really don't enjoy so is there way to disable the loop while using the mic for games like vrchat etc?

**Comment by LamaSlayer14** [comment]
https://youtu.be/_5k9htTdpuI?si=Bo6OjGeSup86h7Gi&t=249
This might fix it, I know I'm VERY late
